### PR TITLE
[IMP] beesdoo_shift: generate task template for multiple shift

### PR DESCRIPTION
**Pull request by tfrancoi**
Allow to choose many planning when you generate task template
from task type

### CODE REVIEW

File: beesdoo_shift/wizard/batch_template.py [hunk 1]
```diff
@@ -28,24 +34,23 @@ class GenerateShiftTemplate(models.TransientModel):
 
     @api.multi
     def generate(self):
-        self.ensure_one()
```
Comment (robinkeunen) [inline beesdoo_shift/wizard/batch_template.py:31]: I'll put the ensure_one back in.

File: beesdoo_shift/wizard/batch_template.py [hunk 2]
```diff
@@ -28,24 +34,23 @@ class GenerateShiftTemplate(models.TransientModel):
...
-        ids = []
```
Comment (robinkeunen) [inline beesdoo_shift/wizard/batch_template.py:32]: and this as well.

Comment (robinkeunen) [inline beesdoo_shift/wizard/batch_template.py:32]: and this as well.

**Review by robinkeunen — APPROVED** [review]
linted and fixed a typo

File: beesdoo_shift/wizard/batch_template.py (hunk 1 above)
Comment (tfrancoi) [reply beesdoo_shift/wizard/batch_template.py:31]: Put all the ensure_one you want

**Review by polchampion — COMMENTED** [review]
Tested on bees-test and approved

**Pull request merged**